pkg/windows/windows.go

```diff
@@ -532,7 +532,7 @@ func (vm *windows) ConfigureHybridOverlay(nodeName string) error {
 	return nil
 }
 
-func (vm *windows) ConfigureCNI(configFile string) error {
+func (vm *windows) EnsureCNI(configFile string) error {
```

Comment:
This can be renamed to `EnsureCNIConfig` as there are other responsibilities such as copying CNI plugins that this function does not handle